src/overlays/actors/ovl_En_Hg/z_en_hg.c

```diff
@@ -165,17 +180,17 @@ void func_80BCF398(EnHg* this, PlayState* play) {
     }
 }
 
-void func_80BCF468(EnHg* this) {
+void EnHg_SetupMoveTowardsPlayer(EnHg* this) {
     Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimationInfo, 1);
-    this->actionFunc = func_80BCF4AC;
+    this->actionFunc = EnHg_MoveTowardsPlayer;
 }
 
-void func_80BCF4AC(EnHg* this, PlayState* play) {
+void EnHg_MoveTowardsPlayer(EnHg* this, PlayState* play) {
```

Comment:
This is fine as is, but thoughts on `ChasePlayer`?

Reply:
That's a more concise way to put it, I agree. Thanks 👍